README.md

```diff
@@ -56,22 +40,24 @@ project(":frameworks/cockroachdb").name = "cockroachdb"
   - [Backup Storage](#backup-storage)
   - [Enterprise Backup and Restore](#enterprise-backup-restore)
 - [Supported Versions](#supported-versions)
+- [Build Instructions](#build-instructions)
 
 <a name="overview"></a>
 # Overview
 
 DC/OS CockroachDB is an automated service that makes it easy to deploy and manage CockroachDB on [DC/OS](https://mesosphere.com/product/).
 
-CockroachDB is a distributed SQL database built on a transactional and
+CockroachDB is an open source distributed SQL database built on a transactional and
 strongly-consistent key-value store. It **scales** horizontally;
 **survives** disk, machine, rack, and even datacenter failures with
 minimal latency disruption and no manual intervention; supports
 **strongly-consistent** ACID transactions; and provides a familiar
 **SQL** API for structuring, manipulating, and querying data.
 
-For more details, check out [its website](https://www.cockroachlabs.com/),
-[FAQ](https://cockroachlabs.com/docs/frequently-asked-questions.html), or
-original [design document](https://github.com/cockroachdb/cockroach#design).
+For more details, check out [our website](https://www.cockroachlabs.com/),
+[FAQ](https://cockroachlabs.com/docs/frequently-asked-questions.html),
```

Comment:
Might as well use the `/docs/stable/frequently-asked-questions.html` url here, too.

Reply:
Done.